core/src/com/fallenflame/game/LightController.java

```diff
@@ -89,7 +89,7 @@ public void initialize(PlayerModel player, JsonValue levelLighting, World world,
         RayHandler.useDiffuseLight(true);
         rayhandler = new RayHandler(world, Gdx.graphics.getWidth(), Gdx.graphics.getHeight());
         rayhandler.setCombinedMatrix(raycamera);
-        rayhandler.setAmbientLight(0, 0, 0, 0);
+        rayhandler.setAmbientLight(0, 0, 0, 0.2f);
```

Comment:
Try to set all constants like this as constants defined at the top of the class so they can be easily found and adjusted.